rollingshutter.py

```diff
@@ -1,31 +1,35 @@
-def rs(folder,speed=1):
-    
-    from PIL import Image
-    import glob
-    
+import glob
+import os.path
+
+from PIL import Image
+
+
+def rolling_shutter(folder, speed=1):
     # ENTER YOUR DIRECTORIES AND FILE TYPE HERE
-    frame_dir = "/Users/mattparker/Documents/rollingshuttervideos/" + str(folder) + "/"
+    frame_dir = os.path.join("/Users/mattparker/Documents/rollingshuttervideos/", str(folder))
```

Comment:
Avoid hardcoding the username?  It would be better to start at the user's Document folder, like: /Users/mattparker/Documents
```python
docs_dir = os.path.join(os.path.expanduser('~'), 'Documents')
frame_dir = os.path.join(docs_dir, "rollingshuttervideos", folder)
# or...
frame_dir = os.path.join(os.path.expanduser('~'), "Documents", "rollingshuttervideos", folder)
```

Reply:
Totally agree. That said, I'm going to resolve this and merge away - given how long this has been open, I feel bad asking for any changes. If you want to PR something like this in the future, you're more than welcome - I'd be happy to accept it 😄